### PR TITLE
+ node_mdns typings + test

### PR DESCRIPTION
**Pull request by reppners**

### CODE REVIEW

File: node_mdns/node_mdns.d.ts [hunk 1]
```diff
@@ -0,0 +1,295 @@
+// Type definitions for node_mdns
+// Project: https://github.com/agnat/node_mdns
+// Definitions by: Stefan Steinhart <https://github.com/reppners>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../node/node.d.ts" />
+
+// interface for extending with custom resolvers
+interface MDNSResolverSequenceTasks {
```
Comment (vvakame) [inline node_mdns/node_mdns.d.ts]: I think this interface move into MDNS module.
http://goo.gl/qiizvX

File: node_mdns/node_mdns.d.ts [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+
+    var kDNSServiceProtocol_IPv4:number;
+    var kDNSServiceProtocol_IPv6:number;
+    var kDNSServiceProtocol_UDP:number;
+    var kDNSServiceProtocol_TCP:number;
+}
+
+declare module "mdns" {
+
+    export = MDNS;
```
Comment (vvakame) [inline node_mdns/node_mdns.d.ts]: please check following sample.
http://goo.gl/bM2oH8
Would you like this tips?

**Comment by vvakame** [comment]
@reppners thank you your contributions.
please rename node_mdns/node_mdns.d.ts to mdns/mdns.d.ts
http://definitelytyped.org/guides/contributing.html#naming-the-file
https://www.npmjs.com/package/mdns

**Comment by reppners** [comment]
done :) i hope it all looks well now. thx for your hint about the module declaration - didn't know that.

**Comment by vvakame** [comment]
@reppners thanks mate!

**Pull request merged**